queries/zathurarc/highlights.scm

```diff
@@ -0,0 +1,36 @@
+; General
+(include_directive
+  (command) @keyword.import)
```

Comment:
Needs to be added **after** `(command) @keyword`

Reply:
Done.